source/_integrations/recorder.markdown

```diff
@@ -8,7 +8,7 @@ ha_release: pre 0.7
 ha_quality_scale: internal
 ---
 
-The `recorder` integration is responsible for storing details in a database, which then are handled by the [`history` integration](/integrations/history/).
```

Comment:
Please do not remove this statement.

Reply:
No idea how that happened. Very sorry! - Readded.